### PR TITLE
update text

### PR DESCRIPTION
**Pull request by Account2Hau**
update text remove

### CODE REVIEW

File: cac-lenh-linux [hunk 1]
```diff
@@ -1,29 +1,5 @@
...
 
-
-##2.Part1: System information
-
-arch hiển thị cấu trúc của máy(1)
-uname -m hiển thị cấu trúc của máy(2)
-uname -r hiển thị phiên bản kernel đang sử dụng
-dmidecode -q Hiển thị hệ thống phần cứng - (SMBIOS / DMI)
-hdparm -i /dev/hda trình bày những đặc trưng của 1 ổ cứng ( lưu ý, hard disk cap ATA là hda còn SATA là sda )
-
```
Comment (nguyennhathau) [inline cac-lenh-linux:11]: remove here

File: cac-lenh-linux [hunk 2]
```diff
@@ -48,9 +24,7 @@ ls hiển thị tập tin và thư mục
 ls -F hiển thị tập tin trong thư mục
 ls -l hiển thị chi tiết tập tin và thư mục
 ls -a hiển thị tập tin ẩn
```
Comment (nguyennhathau) [inline cac-lenh-linux:26]: ok

**Review by nguyennhathau — CHANGES_REQUESTED** [review]

**Comment by Account2Hau** [comment]
done

**Comment by nguyennhathau** [comment]
> done

ok

**Pull request merged**